### PR TITLE
Align rubocop configuration files with updated rules

### PR DESCRIPTION
**Pull request by taltcher**
# Background
Align rubocop configuration files with updated rules

### Project Management
- [x] Review it as needed with the technical team leader?
- [x] Attach the PR to the Trello card?

### PR
- [x] Assign yourself to the PR
- [x] Add at least one reviewer

### Before-Merge
- [ ] Update the repos dependent on those rules - delete the current cached rubocop.yml, and run `rubocop` again to refresh
- [ ] Validate changes against affected repositories, and if needed, prepare PRs to update repositories according to changes

### Post-Merge
- [ ] Verify that your Trello card was moved to today's release and that it was announced in #releases slack channel

### CODE REVIEW

**Comment by taltcher** [comment]
trello::this tech-debt

**Comment by taltcher** [comment]
trello::this tech

File: rubocop.yml [hunk 1]
```diff
@@ -67,14 +73,20 @@ AllCops:
   # by overriding DisplayCopNames, or by giving the `--no-display-cop-names`
   # option.
   DisplayCopNames: true
-  DisplayStyleGuide: true
+  # Style guide URLs are not displayed in offense messages by default. Change
+  # behavior by overriding `DisplayStyleGuide`, or by giving the
+  # `-S/--display-style-guide` option.
+  DisplayStyleGuide: false
```
Comment (selfish) [inline rubocop.yml:79]: Why did this change?

**Review by selfish — DISMISSED** [review]
One comment.
I couldn't really review every line.
I assume this is an automated change, so let's roll. :)

File: rubocop.yml (hunk 1 above)
Comment (taltcher) [reply rubocop.yml:79]: Checking

Comment (taltcher) [reply rubocop.yml:79]: You are right... changing it back

**Comment by taltcher** [comment]
@selfish It was not fully automated :-) I had to manually merge custom definitions we had into their default suggested template

**Review by selfish — APPROVED** [review]

**Pull request merged**